src/appimagetool.c

```diff
@@ -62,10 +62,12 @@ extern unsigned int runtime_len;
 #endif
 
 enum fARCH { 
-    fARCH_i386,
+    fARCH_i386 = 0,
     fARCH_x86_64,
     fARCH_arm,
-    fARCH_aarch64
+    fARCH_aarch64,
+    fARCH_loongarch64,
+    fARCH_SIZE
```

Comment:
These changes are a bit of a hack. They are likely to work, but I would not rely on it. You should at least put a comment here what this entry is there for, and why you added an explicit starting value.

Reply:
done.